src/models/transactions/offerCancel.ts

```diff
@@ -0,0 +1,24 @@
+import { ValidationError } from "../../common/errors"
+import { BaseTransaction, verifyBaseTransaction } from "./common"
+
+export interface OfferCancel extends BaseTransaction {
+    TransactionType: "OfferCancel"
+    OfferSequence: number
+}
+
+/**
+ * Verify the form and type of an OfferCancel at runtime.
+ * 
+ * @param tx - An OfferCancel Transaction
+ * @returns - Void.
+ * @throws - When the OfferCancel is Malformed.
+ */
+ export function verifyOfferCancel(tx: OfferCancel): void {
+    verifyBaseTransaction(tx)
+
+    if (tx.OfferSequence === undefined)
+        throw new ValidationError("OfferCancel: missing field OfferSequence")
+
+    if (typeof tx.OfferSequence !== 'number')
+        throw new ValidationError("OfferCancel: invalid OfferSequence")
```

Comment:
```suggestion
        throw new ValidationError("OfferCancel: OfferSequence must be a number")
```

Reply:
I think `"OfferSequence must be a number"` is a more concise error message than `"OfferCancel: invalid OfferSequence"`